.circleci/config.yml

```diff
@@ -87,128 +65,88 @@ _snyk_options: &snyk_options
       - run:
-          name: Build css
-          command: npm run css-build
+          name: Fake local env file
+          command: echo "" > .env
       - run:
-          name: Minify js and css
-          command: npm run minify-js-css
-      - run:
-          name: Run wiremock
-          command: java -jar wiremock-standalone-3.3.1.jar --global-response-templating --port 9091
+          name: Start X11 virtual framebuffer server (if not running)
+          command: Xvfb :99 || true
           background: true
       - run:
-          name: Run the app
-          command: npm run start-feature
-          background: true
+          name: Start services
+          command: DISPLAY=:99 ENABLE_HEARING_OUTCOMES=true docker compose up -d --wait
       - run:
-          name: Wait for node app to start
-          command: sleep 5
+          name: Log services
+          command: docker compose logs -f -t > services.log
+          background: true
       - run:
           name: Run integration tests
-          halt_build_on_fail: false
-          command: npm run int-test
-      - store_test_results:
-          path: test-results
-      - store_artifacts:
-          path: integration-tests/videos
+          command: docker exec -it prepare-a-case-cypress cypress run --browser chrome --config-file=cypress/cypress.config.js
       - store_artifacts:
-          path: integration-tests/screenshots
-
-  pact:
+          path: integration-tests/output
+      - store_artifacts:    
+          path: services.log
+  pact_push:
     environment:
       PACT_BROKER_DISABLE_SSL_VERIFICATION: "true"
     executor: builder
     steps:
       - checkout
-      - mem/recall:
-          env_var: APP_VERSION
-      - attach_workspace:
-          at: ~/app
-      - restore_cache:
-          key: dependencies-cache-{{ checksum "package.json" }}
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build Development Image
+          command: docker build --target=ci -t prepare-a-case-app -f Dockerfile .
       - run:
           name: Generate pact files
           halt_build_on_fail: false
-          command: npm run pact-test
+          command: docker run -it prepare-a-case-app /usr/src/app/node_modules/.bin/jest --config=./tests/jest.pact.config.js --ci
       - store_artifacts:
           path: 'pact/pacts'
       - run:
           name: Upload pact files to broker
-          command: |
-            npx pact-broker publish pact/pacts \
-              --broker-base-url $PACT_BROKER_BASE_URL \
-              --consumer-app-version $CIRCLE_SHA1 --tag $CIRCLE_BRANCH \
-              -u $PACT_BROKER_USERNAME -p $PACT_BROKER_PASSWORD
-
-  build_docker:
+          command: docker run -it prepare-a-case-app pact-broker publish pact/pacts --broker-base-url $PACT_BROKER_BASE_URL --consumer-app-version $CIRCLE_SHA1 --tag $CIRCLE_BRANCH -u $PACT_BROKER_USERNAME -p $PACT_BROKER_PASSWORD"
```

Comment:
not sure running `pact-broker` in docker container by itself is valid, `npx` has been removed (unless you've added that as a dep in package.json?)

Reply:
Quite right but pact is currently disabled, it's in the commit message at the bottom.